core/src/tangram.cpp

```diff
@@ -104,6 +104,11 @@ void update(float _dt) {
 
     m_view->update();
 
+    // Set time uniforms for all style's shader programs
+    for (auto& style : m_scene->styles()) {
+        style->getShaderProgram()->setUniformf("u_time", _dt);
```

Comment:
This also might be best in the render loop, we have the update/render in the main loop, but it might be safe to simply let update not involve any gl call if possible.